code/site/components/com_pages/model/entity/page.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * Joomlatools Pages
+ *
+ * @copyright   Copyright (C) 2018 Johan Janssens and Timble CVBA. (http://www.timble.net)
+ * @license     GNU GPLv3 <http://www.gnu.org/licenses/gpl.html>
+ * @link        https://github.com/joomlatools/joomlatools-framework-pages for the canonical source repository
+ */
+
+class ComPagesModelEntityPage extends KModelEntityAbstract
+{
+    public function __construct(KObjectConfig $config)
+    {
+        parent::__construct($config);
+
+        //Create the config object (cannot use factory)
+        $config = (new ComPagesObjectConfigPage())->fromFile($this->file);
+
+        //Set the properties
+        $this->setProperties($config->toArray(), false);
+
+        //Se the content
+        $this->_content = $config->getContent();
+    }
+
+    protected function _initialize(KObjectConfig $config)
+    {
+        $config->append(array(
+            'data' => array(
+                'title'       => '',
+                'description' => '',
+                'date'        => '',
+                'published'   => true,
+                'access'      => array(
+                    'roles'  => array('public'),
+                    'groups' => array('public', 'guest'),
+                    'redirect'    => '',
```

Comment:
@johanjanssens,

`redirect`, `metadata`, and etc are inside the `access` array. I think they should be outside.

Reply:
Good catch, typo, fixed it.